src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from "react-dom";
+import element from "./components/reactCreateElementComponent";
+import ReactComponent from "./components/reactComponent";
+import ReactPureComponent from "./components/reactPureComponent";
+import functionalComponent from "./components/functionalComponent";
+
+ReactDOM.render(element, document.getElementById('React.createElement'));
+ReactDOM.render(<ReactComponent/>, document.getElementById('React.Component'));
+ReactDOM.render(<ReactPureComponent/>, document.getElementById('React.PureComponent'));
+ReactDOM.render(functionalComponent, document.getElementById("Functional component"));
```

Comment:
The same, functional component name should be started with uppercase letter and passed as jsx

Reply:
done